Cargo.toml

```diff
@@ -23,4 +23,4 @@ page_table_entry = "0.4.1"
 memory_addr = "0.3.1"
 
 axaddrspace = { git = "https://github.com/arceos-hypervisor/axaddrspace.git" }
-axvcpu = { git = "https://github.com/arceos-hypervisor/axvcpu.git" }
+axvcpu = { git = "https://github.com/arceos-hypervisor/axvcpu.git" }
```

Comment:
what is the modification  here

Reply:
I don't know. This may be due to "cargo fmt".

Reply:
`cargo fmt` will not edit the .toml file, just revert this modification.